conf/ipa.yaml.template

```diff
@@ -15,3 +15,6 @@ IPA:
   GROUPS:
     - satadmins
     - satusers
+  IDM_PASSWORD:
+  IDM_SERVER_IP:
```

Comment:
`IDM_SERVER_IP` = forward resolved `IPA:` -> `HOSTNAME:`

So I suggest you to drop this setting and use this instead:
```
socket.gethostbyname(settings.ipa.hostname)
```